packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -115,14 +115,6 @@ class MenuContent extends React.Component {
     return { isSelectableMenu: this.isSelectable() };
   }
 
-  componentWillReceiveProps(newProps) {
```

Comment:
I don't believe this was doing anything impactful. I tested removing and went through open and closing nested menus with mouse / keyboard and didn't notice any impact. If we think this should be preserved, let me know and I can update it.

Reply:
I figured out why this is needed. Going to incorporate it into the refactor. Thank goodness for visual regression tests.